julia_src/reopt_model.jl

```diff
@@ -514,19 +514,27 @@ function add_storage_op_constraints(m, p)
 		<= p.ProductionFactor[t,ts] * p.LevelizationFactor[t] * m[:dvRatedProduction][t,ts]
 	)
 	# Constraint (4f)-1: (Hot) Thermal production sent to storage or grid must be less than technology's rated production
+	# Constraint (4f)-1a: BoilerTechs
 	if !isempty(p.BoilerTechs)
 		if !isempty(p.SteamTurbineTechs)
-            @constraint(m, HeatingTechProductionFlowCon[b in p.HotTES, t in p.BoilerTechs, ts in p.TimeStep],
+            @constraint(m, BoilerTechProductionFlowCon[b in p.HotTES, t in p.BoilerTechs, ts in p.TimeStep],
                     m[:dvProductionToStorage][b,t,ts] + m[:dvThermalToSteamTurbine][t,ts]  <=
                     p.ProductionFactor[t,ts] * m[:dvThermalProduction][t,ts]
                     )
         else
-            @constraint(m, HeatingTechProductionFlowCon[b in p.HotTES, t in p.BoilerTechs, ts in p.TimeStep],
+            @constraint(m, BoilerTechProductionFlowCon[b in p.HotTES, t in p.BoilerTechs, ts in p.TimeStep],
                     m[:dvProductionToStorage][b,t,ts]  <=
                     p.ProductionFactor[t,ts] * m[:dvThermalProduction][t,ts]
                     )
         end
     end
+	# Constraint (4f)-1b: SteamTurbineTechs
```

Comment:
@Bill-Becker can you please review this? I was seeing timesteps where SteamTurbine dvProductionToStorage exceeded SteamTurbine dvThermalProduction in a test I was running and realized there wasn't a constraint for SteamTurbineTechs.

Reply:
good!